public/app/features/dashboard/panel_editor/VizTypePicker.tsx

```diff
@@ -87,7 +87,12 @@ export const VizTypePicker: React.FC<Props> = ({ searchQuery, onTypeChange, curr
   return (
     <div className={styles.grid}>
       {hasResults ? (
-        renderList.map((plugin, index) => renderVizPlugin(plugin, index))
+        renderList.map((plugin, index) => {
+          if (plugin.state === PluginState.deprecated) {
```

Comment:
If you use a deprecated panel, will it be visible somewhere else?

Reply:
Nope

Reply:
We need to come up with something here, good catch. In the meantime i'm gonna merge this